### PR TITLE
feat: web components svelte refactor

### PR DESCRIPTION
**Pull request by willmcvay**
Fix: #127 
Fix: #199

### CODE REVIEW

File: packages/web-components/src/search-widget/client/api/constants.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const CONSTANTS: { [key: string]: { [key: string]: string } } = {
+  LOCAL: {
+    WEB_COMPONENT_API_BASE_URL_SEARCH_WIDGET: 'http://localhost:3000',
+  },
+  DEV: {
+    WEB_COMPONENT_API_BASE_URL_SEARCH_WIDGET: 'http://localhost:3000',
+  },
+  PROD: {
+    WEB_COMPONENT_API_BASE_URL_SEARCH_WIDGET: 'http://localhost:3000',
```
Comment (tanphamhaiduong) [inline packages/web-components/src/search-widget/client/api/constants.ts]: So will we change it to ENV later right ?

File: packages/web-components/src/search-widget/client/api/constants.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+const CONSTANTS: { [key: string]: { [key: string]: string } } = {
+  LOCAL: {
+    WEB_COMPONENT_API_BASE_URL_SEARCH_WIDGET: 'http://localhost:3000',
+  },
+  DEV: {
+    WEB_COMPONENT_API_BASE_URL_SEARCH_WIDGET: 'http://localhost:3000',
```
Comment (tanphamhaiduong) [inline packages/web-components/src/search-widget/client/api/constants.ts]: So will we change it to ENV later right ?

File: packages/web-components/package.json [hunk 3]
```diff
@@ -9,53 +9,42 @@
...
+    "serverless-offline": "^5.12.1",
+    "serverless-webpack": "^5.3.1",
+    "sirv-cli": "^0.4.5",
+    "svelte-jester": "^1.0.4",
+    "svelte-preprocess": "^3.4.0"
   },
-  "peerDependencies": {
-    "react": "^16.10.2",
-    "react-dom": "^16.10.2"
+  "dependencies": {
```
Comment (vuhuucuong) [inline packages/web-components/package.json]: you should include `@reapit/cognito-auth` as a dependency here, so yarn build can build in correct order, and also if we want to use a fixed version of it, just provide the version number in here.

Comment (vuhuucuong) [reply packages/web-components/package.json]: another cool thing about this is : hen you make some changes in `@reapit/cognito-auth`, all the tests in all packages which have `@reapit/cognito-auth` as a dependency will also be run by `lerna run test:update-badges`. This make sure our changes in `@reapit/cognito-auth` doesn't break another apps.
The same thing's applied for `@reapit/elements`, which forces you to update snapshot of all apps after making changes in `@reapit/elements`. That's very useful :)

Comment (willmcvay) [reply packages/web-components/package.json]: Cognito Auth is not a dependency of this package and won't be - this is client credentials auth flow so the client side app does not need the user to login

File: packages/web-components/src/search-widget/client/api/constants.ts (hunk 1 above)
Comment (willmcvay) [reply packages/web-components/src/search-widget/client/api/constants.ts]: Updated

File: packages/web-components/src/search-widget/client/api/constants.ts (hunk 2 above)
Comment (willmcvay) [reply packages/web-components/src/search-widget/client/api/constants.ts]: Done!

**Pull request merged**